### PR TITLE
[Module:Tizen] Added routine to get user-id and home-folder @open sesame 02/05 21:09

### PR DESCRIPTION
**Pull request by leemgs**
This commit is to append a routine to calculate automatically user-id
and home-folder that are used by a default account of web-server
(e.g., Apache, Nginx).

Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

---

### CODE REVIEW

**Comment by leemgs** [comment]
@gichan-jang  This PR requires your approval.

File: ci/taos/plugins-base/pr-postbuild-build-tizen.sh [hunk 1]
```diff
@@ -80,27 +82,33 @@ function pr-postbuild-build-tizen-run-queue(){
     # TODO: If you want to support this option in the configuration file for flexibility and maintenance,
     # you may submit a PR in order to put the environment variable (e.g., CUSTOM_GBS_CONF{01|02|03})
     # into the ./config/config-environment.sh.
+    WEB_USER_ID="www-data"
+    HOME_DIR=$( getent passwd "$WEB_USER_ID" | cut -d: -f6 )
```
Comment (gichan-jang) [inline ci/taos/plugins-base/pr-postbuild-build-tizen.sh:86]: Nice!

**Review by gichan-jang — APPROVED** [review]

**Review by anyj0527 — APPROVED** [review]
LGTM✔

File: ci/taos/plugins-base/pr-postbuild-build-tizen.sh [hunk 2]
```diff
@@ -80,27 +82,33 @@ function pr-postbuild-build-tizen-run-queue(){
...
     CUSTOM_GBS_CONF01="./.TAOS-CI/.gbs.conf"
     CUSTOM_GBS_CONF02="./packaging/.gbs.conf"
-    CUSTOM_GBS_CONF03="$HOME/.gbs.conf"
+    CUSTOM_GBS_CONF03="$HOME_DIR/.gbs.conf"
```
Comment (helloahn) [inline ci/taos/plugins-base/pr-postbuild-build-tizen.sh:89]: nice separation.

**Review by helloahn — APPROVED** [review]
LGTM!

**Pull request merged**